### PR TITLE
Updates to PoWEx vocabulary and common vocabularies

### PR DESCRIPTION
**Pull request by kochhar**
Updates
1. Refactored CredentialBase to reference VC and Common vocabularies
2. Updated Common vocabularies to reference schema.org 
3. Updated PoWEx vocabulary to reference CredentialBase and Common vocabularies

### CODE REVIEW

File: common/CertificateBase.jsonld [hunk 1]
```diff
@@ -2,30 +2,35 @@
...
       "@id": "https://github.com/sunbird-specs/vc-specs#Subject",
       "@context": {
         "id": "@id",
         "@version": 1.1,
         "@protected": true,
-        "name": "schema:Text",
-        "mobile": "schema:Text",
-        "email": "schema:Text"
+
+        "sunvc": "https://github.com/sunbird-specs/vc-specs#",
```
Comment (pramodkvarma) [inline common/CertificateBase.jsonld:31]: You can prefix with 'sb' instead of 'sun' as in 'sbvc' instead of 'sunvc'

File: work/powex.jsonld [hunk 2]
```diff
@@ -5,38 +5,32 @@
...
-        "certificateId": "@id",
+
         "issuer": { "@id": "cred:issuer", "@type": "schema:Organisation" },
         "legalName": "schema:legalName",
-        "address": { "@id": "schema:address", "@type": "schema:PostalAddress" },
-        "issuedOn": "cred:issuanceDate",
-        "credentialSubject": { "@id": "cred:credentialSubject", "@type": "schema:Person" },
-        "name": "schema:name",
-        "jobTitle": "schema:jobTitle",
+        "address": { "@id": "schema:address", "@type": "sunvc:Address" },
```
Comment (pramodkvarma) [inline work/powex.jsonld:28]: Some SBVC core objects like 'fullname' starts with small letter while objects like 'Address' starts with capital. Any reason? If it needs fix can you please fix it aa well @kochhar ? Thanks. @dileepbapat

File: work/powex.jsonld [hunk 3]
```diff
@@ -5,38 +5,32 @@
...
+        "credentialSubject": { "@id": "cred:credentialSubject", "@type": ["schema:Person", "sunvc:Subject"] },
+        "lastJobTitle": "schema:jobTitle",
         "worksFor": { "@id": "schema:worksFor", "@type": "schema:Organisation" },
-        "worksAs": { "@id": "powex:worksAs", "@type": "schema:OccupationalRole" },
-        "roleType": { "@id": "powex:roleType", "@type": "string" },
-        "startDate": { "@id": "schema:startDate", "@type": "xsd:dateTime" },
-        "endDate": { "@id": "schema:endDate", "@type": "xsd:dateTime" },
-        "occupationalIndustry": { "@id": "occupationalIndustry", "@type": "string" },
-        "occupationalCategory": { "@id": "occupationalCategory", "@type": "string" },
+        "worksAs": { "@id": "powex:worksAs", "@type": "powex:OccupationalRole" },
```
Comment (pramodkvarma) [inline work/powex.jsonld:33]: Is powex local object for proof of work? Could we reference it as 'sbvc.powex'? @kochhar

**Review by pramodkvarma — CHANGES_REQUESTED** [review]
I have commented during review. Can you please check my comments? @kochhar ? Thanks.

**Comment by pramodkvarma** [comment]
Btw, I moved vc-specs repo into a top level org called INCOMS on Github where all kinds of VC models can be worked out. @kochhar your object namespaces will need update. Can you please do?

**Comment by kochhar** [comment]
Definitely. I will update the references and send a PR

**Comment by pramodkvarma** [comment]
Thanks a lot @kochhar

**Pull request merged**